### ISSUE TITLE
'compat' folder is missing from freescout 1.8.3

### ISSUE THREAD

**Issue by GaneshKandu** [freescout-helpdesk/freescout#1714]
Hi,

I am trying to install freescout 1.8.3 but getting blank page.

```
[Mon Feb 07 03:03:39 2022] [warn] [client 122.177.254.213] PHP Warning:  require_once(compat/vlucas/phpdotenv/src/Loader.php): failed to open stream: No such file or directory in public/install.php on line 29
[Mon Feb 07 03:03:39 2022] [error] [client 122.177.254.213] PHP Fatal error:  require_once(): Failed opening required 'compat/vlucas/phpdotenv/src/Loader.php' in public/install.php on line 29

```

it causing issue in installing

**Comment by freescout-helpdesk** [comment]
See installation instructions: https://github.com/freescout-helpdesk/freescout/releases/tag/1.8.3

**Comment by GaneshKandu** [comment]
> See installation instructions: https://github.com/freescout-helpdesk/freescout/releases/tag/1.8.3

I am doing fresh install 

#### [Method 5. Update manually](https://github.com/freescout-helpdesk/freescout/wiki/Updating-FreeScout#method-5-update-manually)

I am getting same issue on ```https://example.com/tools.php```

``` Warning: require_once(compat/vlucas/phpdotenv/src/Loader.php): failed to open stream: No such file or directory in public/tools.php on line 29 Fatal error: require_once(): Failed opening required 'compat/vlucas/phpdotenv/src/Loader.php' in public/tools.php on line 29 ```



####  [Method 3. Update using GIT](https://github.com/freescout-helpdesk/freescout/wiki/Updating-FreeScout#method-3-update-using-git)

```SQLSTATE[42S02]: Base table or view not found: 1146 Table 'freescout183.ltm_translations' doesn't exist (SQL: select distinct `locale` from ltm_translations)```

---

[8.2 Manual installation](https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#82-manual-installation) This works but [8.1 Using web installer](https://github.com/freescout-helpdesk/freescout/wiki/Installation-Guide#81-using-web-installer) this doesn't.

**Comment by freescout-helpdesk** [comment]
Fixed in 1.8.4 release.

**Comment by GaneshKandu** [comment]
@freescout-helpdesk 
Thanks 👍
